dashboard/config/locales/en.yml

```diff
@@ -1057,15 +1057,18 @@ en:
       email: 'Email'
       email_alternate: 'Alternate Email'
       gender: 'Gender Identity'
+      school: 'School'
       school_name: 'School Name'
       school_city: 'School City'
       school_country: 'School Country'
+      school_department_region: 'School Department/Region'
       colombian_school_city: 'School City'
       colombian_chilean_school_department: 'School Department/Region'
```

Comment:
I think it makes sense to create a follow up task to replace the use of `colombian_chilean_school_department` with the universal `school_department_region` after the next i18n sync. It is a little hairier before the sync.